src/services/Map/param.tsx

```diff
@@ -147,6 +147,8 @@ const legendData: (string | TypeGroupLegend)[] = [
     type: 'list',
   },
   'separator',
+  'statsByArea',
```

Comment:
@cspriet Idéalement ici il faudrait plutôt introduire la notion de "potentiels de raccordement", j'avais fait ça au tout début. Je mets potentialForConnectionModal ?

Reply:
En enlevant le For ça passe aussi je pense. Mais ça me paraît pas mal en anglais effectivement